web-client/public/js/upload.js

```diff
@@ -18,20 +18,6 @@ export const upload = function (sliderObject, sliderGroupController, drawGraph,
     var TOOLTIP_SHOW_DELAY    = 700;
     var TOOLTIP_HIDE_DELAY    = 100;
 
-  // Demo Stuff
```

Comment:
OK, so this is a large diff, but note that it's _all deletions_. That's because we've now fully begun the process of breaking this huge monolith of code apart, separating it into the appropriate M-V-C modules. Eventually, _upload.js_ will cease to exist (I think).